binding/Binding/GRContext.cs

```diff
@@ -105,6 +105,6 @@ public void GetResourceCacheUsage (out int maxResources, out long maxResourceByt
 		public int GetRecommendedSampleCount (GRPixelConfig config, float dpi) => 0;
 
 		internal static GRContext GetObject (IntPtr handle) =>
-			GetOrAddObject (handle, (h, o) => new GRContext (h, o));
+			new GRContext (handle, true);
```

Comment:
Because we _know_ the object will always be new and is never in the dictionary, we can further improve perf by not even using a "object factory". Just use the constructor.